db/versions/0017.yml

```diff
@@ -102,3 +102,63 @@ methods:
           end;
         end if;
       end
+  add_denylist_address:
+    description: Add a denylist address for the taskcluster-notify service.
+    mode: write
+    serviceName: notify
+    args: address_in jsonb
+    returns: void
+    body: |-
+      begin
+        insert into denylisted_notification(notification_type, notification_address, etag)
+        values (
+          (address_in ->> 'notificationType')::text,
+          (address_in ->> 'notificationAddress')::text,
+          public.gen_random_uuid()
+        ) on conflict do nothing;
+      end
+  delete_denylist_address:
+    description: |-
+      Delete a denylist address for the taskcluster-notify service.
+      Returns number of rows deleted (0 or 1).
+    mode: write
+    serviceName: notify
+    args: address_in jsonb
+    returns: integer
+    body: |-
+      begin
+        delete from denylisted_notification where
+        denylisted_notification.notification_type = (address_in ->> 'notificationType')::text and
+        denylisted_notification.notification_address = (address_in ->> 'notificationAddress')::text;
+        if found then
+          return 1;
+        end if;
+        return 0;
+      end
+  all_denylist_address:
+    description: List all denylist addresses for the taskcluster-notify service.
+    mode: read
+    serviceName: notify
+    args: page_size_in integer, page_offset_in integer
+    returns: table (notification_type text, notification_address text)
+    body: |-
+      begin
+        return query select denylisted_notification.notification_type, denylisted_notification.notification_address
+        from denylisted_notification
+        order by 1, 2
```

Comment:
What does `order by 1, 2` mean?.

Reply:
It means to order by the first column (`denylisted_notification.notification_type`), and then the second column (`denylisted_notification.notification_address`).